mesonbuild/dependencies/python.py

```diff
@@ -13,18 +13,95 @@
 # limitations under the License.
 from __future__ import annotations
 
+import json, sysconfig
 from pathlib import Path
-import sysconfig
 import typing as T
 
-from .. import mlog
+from .. import mesonlib, mlog
```

Comment:
and the same here.

Reply:
The thing is... immediately below, we do exactly the same thing with e.g.

```diff
+from .base import process_method_kw, DependencyMethods, DependencyTypeName, ExternalDependency, SystemDependency
```

I really find it more natural and idiomatic to do this, and I don't see the point in spreading imports across one bound name per line. That just becomes difficult to read and difficult to scroll past.

Reply:
Idiomatically modules are imported one per line, objects from modules are imported on a single line:
```python
import os
import sys

from os.path import join, dirname, basename
```
https://peps.python.org/pep-0008/#imports

I get it, style guides always have something in them that annoys people. But having a single styl, makes the whole codebase much more readable.

Reply:
And this is `from foo import bar, baz`...

I'm really unhappy that people keep campaigning to make ugly code, the same people who defend "modules are imported one per line" are defending "objects from modules are imported one per line" and for the same reason.

This is nothing to do with style. I've already stated my ***strong*** objections to "add a lint rule to force objects from modules to be imported one per line because I think it's better for rebasing". What I actually want is a lint rule that forbids that.